### PR TITLE
Improve doc

### PR DESCRIPTION
**Pull request by martavp**
At AU, we have been working on improving the documentation. 

We are still missing a few tasks before this PR is ready to be reviewed, but @Parisra, @Adam-Dvorak1 and @ebbekyhl will take care of them next week

TODOs LIST

[x] Add internal links from some sections to others

[x] Harmonize style in title and subtitle

[x] Review the description of the myopic approach and describe better carbon budget

[x] Finish and review Other industrial sectors

[x] Add description of energy consumption in agriculture

[ ] Create an overview of the rules like in PyPSA-Eur (maybe this can be done in a future step)

### CODE REVIEW

**Comment by martavp** [comment]
We are finally ready for this PR to be reviewed.
I have suggested @energyLS, @lisazeyen and @fneum as reviewers.

This PR will conflict with https://github.com/PyPSA/pypsa-eur-sec/pull/264 because we have included many references to specific lines in the config file that would need to be updated once https://github.com/PyPSA/pypsa-eur-sec/pull/264 is merged.

**Comment by fneum** [comment]
Great, thanks @martavp! We'll prioritise this PR over #264.

**Comment by fneum** [comment]
Current build for this PR: https://pypsa-eur-sec.readthedocs.io/en/martavp-improve-doc/

File: doc/myopic.rst [hunk 1]
```diff
@@ -6,13 +6,13 @@ Myopic transition path
```
Comment (energyLS) [inline doc/myopic.rst:6]: Maybe change the section title to "Myopic foresight scenarios" to be consistent with "Overnight (greenfield) scenarios" and "Perfect foresight scenarios"

File: doc/spatial_resolution.rst [hunk 2]
```diff
@@ -4,55 +4,44 @@
 Spatial resolution
 ##########################################
 
-The default nodal resolution of the model follows the electricity
-generation and transmission model `PyPSA-Eur
-<https://github.com/PyPSA/pypsa-eur>`_, which clusters down the
-electricity transmission substations in each European country based on
-the k-means algorithm. This gives nodes which correspond to major load
-and generation centres (typically cities).
+The default nodal resolution of the model follows the electricity generation and transmission model `PyPSA-Eur <https://github.com/PyPSA/pypsa-eur>`_ , which clusters down the electricity transmission substations in each European country based on the k-means algorithm (See `cluster_network <https://pypsa-eur.readthedocs.io/en/latest/simplification/cluster_network.html#rule-cluster-network>`_ for a complete explanation). This gives nodes which correspond to major load and generation centres (typically cities).
```
Comment (energyLS) [inline doc/spatial_resolution.rst:7]:
```suggestion
The default nodal resolution of the model follows the electricity generation and transmission model `PyPSA-Eur <https://github.com/PyPSA/pypsa-eur>`_, which clusters down the electricity transmission substations in each European country based on the k-means algorithm (See `cluster_network <https://pypsa-eur.readthedocs.io/en/latest/simplification/cluster_network.html#rule-cluster-network>`_ for a complete explanation). This gives nodes which correspond to major load and generation centres (typically cities).
```

File: doc/spatial_resolution.rst [hunk 3]
```diff
@@ -4,55 +4,44 @@
...
+The total number of nodes for Europe is set in the config.yaml file under `clusters <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L20>`_.  The number of nodes can vary between 37, the number of independent countries/ synchronous areas, and several hundred. With 200-300 nodes, the model needs 100-150 GB RAM to solve with a commercial solver like Gurobi.
+Not all of the sectors are at the full nodal resolution, and some demand for some sectors is distributed to nodes using heuristics that need to be corrected. Some networks are copper-plated to reduce computational times.
 
-Electricity residential and commercial demand: nodal, distributed in
-each country based on population and GDP.
+Here are some examples of how spatial resolution is set for different sectors in PyPSA-Eur-Sec:
 
-Electricity demand in industry: based on the location of industrial
-facilities from `HotMaps database <https://gitlab.com/hotmaps/industrial_sites/industrial_sites_Industrial_Database>`_.
+•	Electricity network: Modeled as nodal
```
Comment (energyLS) [inline doc/spatial_resolution.rst:22]:
```suggestion
•	Electricity network: Modeled as nodal.
```

File: doc/spatial_resolution.rst [hunk 4]
```diff
@@ -4,55 +4,44 @@
...
-gas to hydrogen is to be considered, the methane grid can be nodally resolved
-based on SciGRID_gas data.
+•	Electricity demand in industry: Modeled as nodal, based on the location of industrial facilities from HotMaps database.
 
-Solid biomass: choice between single node for Europe and nodal where biomass
-potential is regionally disaggregated (currently given per country,
-then distributed by population density within)
-and transport of solid biomass is possible.
+•	Industry demand (heat, chemicals, etc.) : Modeled as nodal, distributed in each country based on locations of industry from HotMaps database.
+•	Hydrogen network: Modeled as nodal (if activated in the `Config <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L260>`_ file).
```
Comment (energyLS) [inline doc/spatial_resolution.rst:33]:
```suggestion
•	Hydrogen network: Modeled as nodal (if activated in the `config <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L260>`_ file).
```

**Review by energyLS — APPROVED** [review]
Great effort. thanks! Corrected mostly only minor grammar/typo/punctuation issues. Did not review "supply_demand.rst" yet, will be done in a follow-up review.

**Comment by fneum** [comment]
Thanks @martavp et al. for preparing this documentation boost, and thanks @energyLS for kicking of the review. One issue I saw in the preview (https://pypsa-eur-sec.readthedocs.io/en/martavp-improve-doc/) is that the equations are not properly resolved. I.e. in RST syntax, math mode needs to be activated not with `$$ ... $$` but with 

```rst
.. math::
   \frac{ \sum_{t=0}^{N}f(t,k) }{N}
```

or inline

```rst
:math:`\frac{ \sum_{t=0}^{N}f(t,k) }{N}`
```

File: doc/myopic.rst [hunk 5]
```diff
@@ -6,13 +6,13 @@ Myopic transition path
...
 
-The transport sector and industry are not affected by the myopic code. In essence, the electrification of road and rail transport, the percentage of electric vehicles that allow demand-side management and vehicle-to-grid services, and the transformation in the different industrial subsectors do not evolve with time. They are kept fixed at the values specified in the configuration file. Including the transport sector and industry in the myopic code is planned for the near future.
-
-See also other `outstanding issues <https://github.com/PyPSA/pypsa-eur-sec/issues/19#issuecomment-678194802>`_.
+The following subjects within the land transport and biomass currently do not evolve with the myopic approach:
+- The percentage of electric vehicles that allow demand-side management and vehicle-to-grid services
```
[the remark this answers is not in the document]
Comment (martavp) [reply doc/myopic.rst]: Thanks, I have made them bullet points here: https://github.com/PyPSA/pypsa-eur-sec/pull/265/commits/7cc336e89e95bf9b06d6d8001cf57d4dd853ef03

**Comment by martavp** [comment]
> 

Thanks for the initial revision @energyLS

**Comment by martavp** [comment]
> Thanks @martavp et al. for preparing this documentation boost, and thanks @energyLS for kicking of the review. One issue I saw in the preview (https://pypsa-eur-sec.readthedocs.io/en/martavp-improve-doc/) is that the equations are not properly resolved. I.e. in RST syntax, math mode needs to be activated not with `$$ ... $$` but with
> 
> ```
> .. math::
>    \frac{ \sum_{t=0}^{N}f(t,k) }{N}
> ```
> 
> or inline
> 
> ```
> :math:`\frac{ \sum_{t=0}^{N}f(t,k) }{N}`
> ```

You are right, we'll soon update this.

File: doc/supply_demand.rst [hunk 6]
```diff
@@ -35,196 +35,579 @@ Also unlike PyPSA-Eur, PyPSA-Eur-Sec subtracts existing electrified heating from
...
+					
+Statistics for the production of ammonia, which is commonly used as a fertilizer, are taken from the `USGS <https://www.usgs.gov/media/files/nitrogen-2017-xlsx>`_ for every country. Ammonia can be made from hydrogen and nitrogen using the Haber-Bosch process.
+					
+$$
+N_2 + 3H_2 → 2NH_3
+$$
+
+
+The Haber-Bosch process is not explicitly represented in the model, such that demand for ammonia enters the model as a demand for hydrogen ( 6.5 MWh $_{H_2}$ / t $_{NH_3}$ ) and electricity ( 1.17 MWh $_{el}$ /t $_{NH_3}$ ) (see `Wang et. al <https://doi.org/10.1016/j.joule.2018.04.017>`_). Today, natural gas dominates in Europe as the source for the hydrogen used in the Haber-Bosch process, but the model can choose among the various hydrogen supply options described in the hydrogen section (see :ref:`Hydrogen supply`)
```
Comment (energyLS) [inline doc/supply_demand.rst:451]:
```suggestion
The Haber-Bosch process is not explicitly represented in the model, such that demand for ammonia enters the model as a demand for hydrogen ( 6.5 MWh $_{H_2}$ / t $_{NH_3}$ ) and electricity ( 1.17 MWh $_{el}$ /t $_{NH_3}$ ) (see `Wang et. al <https://doi.org/10.1016/j.joule.2018.04.017>`_). Today, natural gas dominates in Europe as the source for the hydrogen used in the Haber-Bosch process, but the model can choose among the various hydrogen supply options described in the hydrogen section (see :ref:`Hydrogen supply`).
```

File: doc/supply_demand.rst [hunk 7]
```diff
@@ -35,196 +35,579 @@ Also unlike PyPSA-Eur, PyPSA-Eur-Sec subtracts existing electrified heating from
...
+the percentage that is chemically recycled, see `config file <https://github.com/PyPSA/pypsa-eur-sec/blob/776596ab9ac6a6cc93422ccfd0383abeffb0baa9/config.default.yaml#L316>`_ The energy consumption for those recycling processes are respectively 0.547 MWh $_{el}$/t of HVC (as indicated in the `config file <https://github.com/PyPSA/pypsa-eur-sec/blob/776596ab9ac6a6cc93422ccfd0383abeffb0baa9/config.default.yaml#L318>`_) (`Meys et al. (2020) <https://doi.org/10/gmxv6z>`_), and 6.9 MWh $_{el}$/t of HVC (as indicated in the config file `<https://github.com/PyPSA/pypsa-eur-sec/blob/776596ab9ac6a6cc93422ccfd0383abeffb0baa9/config.default.yaml#L319>`_) based on pyrolysis and electric steam cracking (see `Materials Economics <https://materialeconomics.com/publications/industrial-transformation-2050>`_ report).
```
Comment (energyLS) [inline doc/supply_demand.rst:461]:
```suggestion
the percentage that is chemically recycled, see `config file <https://github.com/PyPSA/pypsa-eur-sec/blob/776596ab9ac6a6cc93422ccfd0383abeffb0baa9/config.default.yaml#L316>`_. The energy consumption for those recycling processes are respectively 0.547 MWh $_{el}$/t of HVC (as indicated in the `config file <https://github.com/PyPSA/pypsa-eur-sec/blob/776596ab9ac6a6cc93422ccfd0383abeffb0baa9/config.default.yaml#L318>`_) (`Meys et al. (2020) <https://doi.org/10/gmxv6z>`_), and 6.9 MWh $_{el}$/t of HVC (as indicated in the config file `<https://github.com/PyPSA/pypsa-eur-sec/blob/776596ab9ac6a6cc93422ccfd0383abeffb0baa9/config.default.yaml#L319>`_) based on pyrolysis and electric steam cracking (see `Materials Economics <https://materialeconomics.com/publications/industrial-transformation-2050>`_ report). 
```

File: doc/supply_demand.rst [hunk 8]
```diff
@@ -35,196 +35,579 @@ Also unlike PyPSA-Eur, PyPSA-Eur-Sec subtracts existing electrified heating from
...
+
+Energy demands for the agriculture, forestry and fishing sector per country are taken from the `JRC IDEES database <https://op.europa.eu/en/publication-detail/-/publication/989282db-ad65-11e7-837e-01aa75ed71a1/language-en>`_.  Missing countries are filled with `eurostat data <https://ec.europa.eu/eurostat/web/energy/data/energy-balances>`_. Agricultural energy demands are split into electricity (lighting, ventilation, specific electricity uses, electric pumping devices), heat (specific heat uses, low enthalpy heat) machinery oil (motor drives, farming machine drives, diesel-fueled pumping devices). Heat demand is for this sector is classified as services rural heat. Time series for demands are assumed to be constant and distributed inside countries in proportion to population.
```
Comment (energyLS) [inline doc/supply_demand.rst:515]:
```suggestion
```
This is a duplicate and described in the subsequent paragraph

File: doc/supply_demand.rst [hunk 9]
```diff
@@ -35,196 +35,579 @@ Also unlike PyPSA-Eur, PyPSA-Eur-Sec subtracts existing electrified heating from
...
+
+For BEV, smart charging is an option. A `certain share <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L172>`_ of the BEV fleet can shift their charging time. The BEV state of charge is forced to be higher than a `set percentage <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L163>`_, e.g. 75%, every day at a `specified hour <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L164>`_, e.g., 7 am, to ensure that the batteries are sufficiently charged for peak usage in the morning and they not behave as seasonal storage. They also have the option to participate in vehicle-to-grid (V2G) services to facilitate system operation if that `is enabled <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L179>`_
```
Comment (energyLS) [inline doc/supply_demand.rst:542]:
```suggestion
For BEV, smart charging is an option. A `certain share <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L172>`_ of the BEV fleet can shift their charging time. The BEV state of charge is forced to be higher than a `set percentage <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L163>`_, e.g. 75%, every day at a `specified hour <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L164>`_, e.g., 7 am, to ensure that the batteries are sufficiently charged for peak usage in the morning and they not behave as seasonal storage. They also have the option to participate in vehicle-to-grid (V2G) services to facilitate system operation if that `is enabled <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L179>`_.
```

File: doc/supply_demand.rst [hunk 10]
```diff
@@ -35,196 +35,579 @@ Also unlike PyPSA-Eur, PyPSA-Eur-Sec subtracts existing electrified heating from
...
+
+**Aviation**
+
+The `demand for aviation <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/scripts/prepare_sector_network.py#L2193>`_ includes international and domestic use. It is modelled as an oil demand since aviation consumes kerosene. This can be produced synthetically or have fossil-origin (see :ref:`Oil-based products supply`).
+
+.. _Shipping:
+
+**Shipping**
+
+Shipping energy demand is covered by a combination of oil and hydrogen. Other fuel options, like methanol or ammonia, are currently not included in PyPSA-Eur-Sec.The share of shipping that is assumed to be supplied by hydrogen can be selected in the `config file <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L198>`_.
```
Comment (energyLS) [inline doc/supply_demand.rst:568]:
```suggestion
Shipping energy demand is covered by a combination of oil and hydrogen. Other fuel options, like methanol or ammonia, are currently not included in PyPSA-Eur-Sec. The share of shipping that is assumed to be supplied by hydrogen can be selected in the `config file <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L198>`_.
```

File: doc/supply_demand.rst [hunk 11]
```diff
@@ -35,196 +35,579 @@ Also unlike PyPSA-Eur, PyPSA-Eur-Sec subtracts existing electrified heating from
...
+•	Hydrogen production by SMR  
+
+•	CHP plants using biomass or methane
+
+•	`Coal power plants <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L242>`_.
+
+Point source emissions are captured assuming a capture rate, e.g. 90%, which can be specified in the `config file <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L249>`_. The electricity and heat demand of process emission carbon capture
+is currently ignored.
+
+DAC (if `included <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L243>`_) includes the adsorption phase where electricity and heat consumptionsare required to assist the adsorption process and regenerate the adsorbent. It also includes the drying and compression of $CO_2$ prior to storage which consumes electricity and rejects heat.
```
Comment (energyLS) [inline doc/supply_demand.rst:600]:
```suggestion
DAC (if `included <https://github.com/PyPSA/pypsa-eur-sec/blob/3daff49c9999ba7ca7534df4e587e1d516044fc3/config.default.yaml#L243>`_) includes the adsorption phase where electricity and heat consumptions are required to assist the adsorption process and regenerate the adsorbent. It also includes the drying and compression of $CO_2$ prior to storage which consumes electricity and rejects heat. 
```

**Comment by fneum** [comment]
Great! Thanks for all your work! The references to lines in the config.yaml will be hard to maintain, but we can find a solution separately from this PR.

**Comment by martavp** [comment]
> Finished the review of `supply_demand.rst`. Great effort, detailed descriptions, well readable. The suggestions are mainly typos. The cross-referencing to subsections seems not to work, please doublecheck. The industry section was tiring to read since it is very detailed, but that's adequate and how it should be :)

thanks for the review, @energyLS Yes, industry always super long to describe :)

**Comment by martavp** [comment]
> Great! Thanks for all your work! The references to lines in the config.yaml will be hard to maintain, but we can find a solution separately from this PR.

Sure, we can find other ways to do it. I think it always clarifies to have a direct link between the documentation and where to change a particular assumption.

**Pull request merged**